source/docs/software/commandbased/convenience-features.rst

```diff
@@ -453,6 +453,20 @@ The ``deadlineWith()`` `decorator <https://first.wpi.edu/FRC/roborio/release/doc
    // Will be a parallel deadline group containing fooCommand, barCommand, and bazCommand; fooCommand is the deadline
    button.whenPressed(fooCommand.deadlineWith(barCommand, bazCommand));
 
+withName (Java only)
+^^^^^^^^^^^^^^^^^^^^
+
+.. note:: This decorator is not supported in C++ due to technical constraints - users should set the name of the command inside their command class instead.
+
+.. todo:: Add link to release documentation when release is made.
```

Comment:
Is it a different link then this? https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj2/command/CommandBase.html#withName(java.lang.String)

Reply:
No, that is correct.